### PR TITLE
reset_password

### PR DESCRIPTION
**Pull request by phancuong1234**

### CODE REVIEW

**Comment by phancuong1234** [comment]
ready

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -34,6 +34,7 @@
                     <a href="redirect/twitter" class="btn btn-primary btn-lg1 btn-block tw"><i class="fa fa-twitter tab"></i> @lang('auth.logintw')</a>
                 </div>
                 <p>@lang('auth.linkregister') <a href="{{route('register.index')}}" title='@lang('auth.register')' >@lang('auth.register')</a></p>
+            <p>Ban Quen Mat Khau <a href="{{route('forgot.index')}}" title='@lang('auth.register')' >gui mail</a></p>
```
Comment (tranducquoc) [inline resources/views/auth/login.blade.php]: i18n

File: resources/views/auth/mailfb.blade.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<p>Mat Khau Moi Cua Tai Khoan <strong>{{$username}}</strong> Co Email:<strong>{{$email}}</strong> La: <strong>{{$newpass}}</strong></p>
+
+
```
Comment (tranducquoc) [inline resources/views/auth/mailfb.blade.php]: remove 2 line này đi em

**Comment by phancuong1234** [comment]
ready

**Comment by tranducquoc** [comment]
commented
conflict

**Comment by phancuong1234** [comment]
ready

**Comment by tranducquoc** [comment]
commented
conflict

**Comment by phancuong1234** [comment]
ready

**Pull request merged**